projects/ngrx.io/content/guide/store/selectors.md

```diff
@@ -266,6 +266,27 @@ export class AppComponent {
 }
 </code-example>
 
+When omitting a generic for the Store, only selectors which operate on `object`s are allowed in `store.select`.
+
+For instance, when using `createFeatureSelector`, the `AppState` argument should be omitted:
+
+<code-example header="index.ts">
+import { createSelector, createFeatureSelector } from '@ngrx/store';
+
+export const featureKey = 'feature';
+
+export interface FeatureState {
+  counter: number;
+}
+
+export const selectFeature = createFeatureSelector<FeatureState>(featureKey);
+
+export const selectFeatureCount = createSelector(
+  selectFeature,
+  (state: FeatureState) => state.counter
```

Comment:
the type of `state` will be inferred from the feature selector

```suggestion
  (state) => state.counter
```

Reply:
yeah, the many of the examples in this document needlessly explicitly declare the type of projector arguments. i can remove it- i was unsure.